src/Commands/ArtCommand.php

```diff
@@ -19,6 +25,24 @@ public function art($name) {
     }
 
     /**
+     * If the user does not specify the $name parameter, then we will
+     * prompt for it here.
+     *
+     * @hook interact
```

Comment:
This would be fully-specified as `@hook interact art`.  If the command name is left off, then the hook is applied to every command in the same class that the hook is in.